tests/v2/codecoverage/agent/main.go

```diff
@@ -77,19 +77,11 @@ func runAgent(ctx context.Context) {
 		if os.Getenv("CLUSTER_CLEANUP") == "true" {
 			err = clean.Cluster()
 		} else if os.Getenv("BINDING_CLEANUP") == "true" {
-			var bindingErr error
-			err = clean.DuplicateBindings(nil)
-			if err != nil {
-				bindingErr = multierror.Append(bindingErr, err)
-			}
-			err = clean.OrphanBindings(nil)
-			if err != nil {
-				bindingErr = multierror.Append(bindingErr, err)
-			}
-			err = clean.OrphanCatalogBindings(nil)
-			if err != nil {
-				bindingErr = multierror.Append(bindingErr, err)
-			}
+			bindingErr := errors.Join(
```

Comment:
Couldn't you simplify this to `err = errors.Join( ... )` rather than creating a new local?

Reply:
Good catch! Changed